doc/api/worker_threads.md

```diff
@@ -4,8 +4,8 @@
 
 > Stability: 1 - Experimental
 
-The `worker_threads` module enables the use of threads with message channels
-between them. To access it:
+The `worker_threads` module enables the use of threads that execute JS code
+in parallel, with message channels between them. To access it:
```

Comment:
Nit: Maybe drop the message channels stuff to focus on The One Thing you want someone to know about the module?:

```markdown
The `worker_threads` module enables the use of threads that execute code in parallel.
```

Reply:
done!